.ci_support/migrations/cuda120.yaml

```diff
@@ -69,13 +73,13 @@ cuda_compiler_version:         # [linux and os.environ.get("CF_CUDA_ENABLED", "F
   - 12.0                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 c_compiler_version:            # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12                         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 11                         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
This is just reapplying the change from commit: https://github.com/conda-forge/pp-sketchlib-feedstock/pull/40/commits/68d36e69735c94bdbc826b0a9c73904fd2c15bc0

Reverting this once the pybind11 + CUDA issues are worked out is tracked in issue: https://github.com/conda-forge/pp-sketchlib-feedstock/issues/43